docs/guides/chip_tool_guide.md

```diff
@@ -1391,3 +1391,180 @@ $ ./chip-tool any read-by-id <cluster-ids> <attribute-ids> <destination-id> <end
     ```
     ./chip-tool any read-by-id 0xFFFFFFFF 0xFFFFFFFF 1 0xFFFF
     ```
+
+<hr>
+
+## Saving users and credentials on door lock devices
```

Comment:
Should this be in the main chip-tool guide?  Or should it be a separate guide that is linked from here?  Seems like we might want some per-complicated-device guides...

Reply:
Fair point. I suggest we wait until there are more sections like this and then start creating device type guides, because it will affect more than just the CHIP Tool Guide. Separating this content from the CHIP Tool Guide now could lead to having two CHIP Tool guides in parallel. 

Would you have suggestions what types of devices might require separate guides?

Reply:
Thermostats seem like an obvious one that might be complicated.

And maybe bridges.